image.go

```diff
@@ -98,19 +97,15 @@ func (i *image) Unpack(ctx context.Context, snapshotterName string) error {
 	return nil
 }
 
-func (i *image) getLayers(ctx context.Context) ([]rootfs.Layer, error) {
+func (i *image) getLayers(ctx context.Context, ps string) ([]rootfs.Layer, error) {
```

Comment:
Nit: Can we get something more explicit than `ps`? It usually makes me think of the `ps` command and hence `processes`

Reply:
platformSpecifier would be more explicit, maybe platform or psp ...